docs/sources/developers/plugins/sign-a-plugin.md

```diff
@@ -32,7 +32,16 @@ To verify ownership of your plugin, you need to generate an API key that you'll
 
 Public plugins need to be reviewed by the Grafana team before you can sign them.
 
-1. Submit your plugin for [review]({{< relref "publish-a-plugin/#publishing-your-plugin-for-the-first-time" >}})
+1. Submit your plugin for review. 
+
+   1. Sign in to [grafana.com](https://grafana.com)
+   1. On the My Account page, go to Org Settings.
```

Comment:
**Bold** UI elements.

Reply:
Not fiinding anything specific on bold in the Grafana style guide, I used MSFT style guide which is to bold UI names only when they are clicked upon. Did I miss something?

Reply:
Issue should be resolved with latest commit.